tasks/get_current_archive_list/API.md

```diff
@@ -165,7 +165,7 @@ Columns used by our database are given standardized names.
   
 - `Returns` - A string representing SQL columns to create.
   Columns required for import but unused by orca will be filled in with `junk` values.
-  For example, 'orca_archive_location text, key_path text, size_in_bytes bigint, last_update timestamptz, etag text, storage_class text, junk7 text, junk8 text, junk9 text, junk10 text, junk11 text, junk12 text, junk13 text, junk14 text'
+  For example, 'orca_archive_location text, key_path text, size_in_bytes text, last_update timestamptz, etag text, storage_class text, junk7 text, junk8 text, junk9 text, junk10 text, junk11 text, junk12 text, junk13 text, junk14 text'
```

Comment:
I would expect size_in_bytes to be of type bigint since it is a number not of type text. Is this a typo or something you are running across while loading? If it is an issue with load, it could be the driver is choosing the wrong type and we could force it into a number.